ndsl/boilerplate.py

```diff
@@ -0,0 +1,95 @@
+from typing import Tuple
+
+import numpy as np
+
+from ndsl import (
+    CompilationConfig,
+    DaceConfig,
+    DaCeOrchestration,
+    GridIndexing,
+    NullComm,
+    QuantityFactory,
+    RunMode,
+    StencilConfig,
+    StencilFactory,
+    SubtileGridSizer,
+    TileCommunicator,
+    TilePartitioner,
+)
+
+
+def _get_one_tile_factory(
```

Comment:
Might rename this as `get_factories_one_tile` and similarly for the others, also to expose it directly?

Reply:
Made more generic and detailed why we don't expose it (simplification & the fact that the orchestration is broken between two inputs instead of one, see #46)